build/_k-restore.shade

```diff
@@ -4,7 +4,7 @@ k-restore
     Restores nuget packages required for k projects. Downloads and executes k sdk tools.
 */}
 
-default restore_options='${E("KPM_restore_options")}'
+default restore_options='${E("NUGET3_restore_options")}'
```

Comment:
The name is entirely used internally. Not consumed by customers. They can be renamed anytime.